wstakemgr/stake.go

```diff
@@ -33,12 +33,12 @@ import (
 	"github.com/decred/dcrutil"
 	walletchain "github.com/decred/dcrwallet/chain"
 	"github.com/decred/dcrwallet/waddrmgr"
+	"github.com/decred/dcrwallet/wallet/txrules"
 	"github.com/decred/dcrwallet/walletdb"
 )
 
 const (
-	revocationFeeTestNet int64 = 10000
-	revocationFeeMainNet int64 = 7500000
+	revocationFeePerKB int64 = 1e6
```

Comment:
Since this is only used in one place below, and that place includes a type conversion to dcrutil.Amount, you might as well just make it a dcrutil.Amount here.